test_data/network_params.yaml

```diff
@@ -1,21 +1,23 @@
 participants:
   - el_type: geth
     cl_type: lighthouse
-    count: 2
+    validator_count: 32
+  - el_type: geth
+    cl_type: prysm
+    validator_count: 32
```

Comment:
why this change?

Reply:
I wanted to have at least a different CL client for variety (to make sure other CLs receive and accept our blocks). Do you prefer to add other CL client/s instead of this one?

Regarding the validator count, default is 64, I lowered it to 32 to speed up the block proposal check (because I thought it was a per-validator thing due to variable naming in the definition of the check when in fact it was a per-client one). I could revert the validator count change if preferred and check times, I don't think it'll impact.

Reply:
Im fine with more CL clients but I thought it would affect the job time (more clients, more compilation right?). Anyway, not a big deal since it seems we're good time wise

Reply:
It will affect kurtosis setup for sure when starting the client (not by a lot as far as I could see) and yes probably also affect some of the checks, but fortunately it wasn't so big of an impact.